.github/workflows/cli-test-prepare-packages.yaml

```diff
@@ -0,0 +1,110 @@
+name: Prepare and call the CLI testing - stub
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      zksolc:
+        type: string
+        description: "zkSolc version, (repo: https://github.com/matter-labs/zksolc-bin/raw/main) e.g. v1.3.22"
+        required: true
+        default: "v1.3.22"
```

Comment:
Let's omit `v` to be consistent with our version tags.

Reply:
done